karma.common.conf.js

```diff
@@ -32,6 +32,9 @@ var webpackTestConfig = {
             // Assume test files are ES6
             {test: /\.test\.js$/, loader: 'babel-loader'},
         ])
+    },
+        node: {
+       fs: 'empty'
```

Comment:
What does this do?

Reply:
The added lines serves the same function for our JS tests as it does in `wepack.common.config.js` as a workaround to suppress an error due to a known webpack issue.